docs/current-1.1/includes/warning.md

```diff
@@ -0,0 +1,4 @@
+> [!WARNING]
```

Comment:
preferably in the future, we'd have a single folder for all includes. my preference would be to put all tokens under docs/includes